src/Components/GSignInButton.js

```diff
@@ -41,6 +48,9 @@ const GSignInButton = props => {
       await GoogleSignin.hasPlayServices();
       await GoogleSignin.signIn();
       setIsSignedIn(true);
+      // TODO: Use generated mnemonic
```

Comment:
We need to use the mnemonic for getting the user wallet, so I marked as TODO, because we need to store the mnemonic somewhere when we decide what state management will be using.